README.md

```diff
@@ -1,2 +1,25 @@
 # embedded
 Mainflux Embedded Components
+
+This repo contains code to connect embedded devices to mainflux.
+
+## Support
```

Comment:
There should be Usage as a title

Reply:
Under Usage completely describe buld process, or refer to individual readme sections in different dirs that explain development